bindings/python/pymongoarrow/api.py

```diff
@@ -306,9 +384,13 @@ def _tabular_generator(tabular):
         for i in tabular.to_batches():
             for row in i.to_pylist():
                 yield row
-    elif DataFrame is not None and isinstance(tabular, DataFrame):
+    elif pd.DataFrame is not None and isinstance(
+        tabular, pd.DataFrame
+    ):  # todo how could DataFrame be None?
```

Comment:
That was when pandas was an optional dependency

Reply:
Safe to remove then?

Reply:
Yep!